### DISCUSSION TITLE
Regen complète de l'énergie au passage d'un niveau

### DISCUSSION THREAD

**Discussion by Mura667** [Crownicles/Crownicles#1966] [Suggestions fermées]
Bonsoir, je trouverais bien et pas trop abusé que quand on passe d'un niveau notre énergie soit complètement régénérée.

**Comment by BastLast** [comment]
Excepté que ca pourrait être abusé avec des gens qui économisent l'xp pr opti le nb de fight max / semaine et c'est pas vraiment le genre d'optimisation que je souhaite encourager.
